rllib/agents/trainer.py

```diff
@@ -347,8 +347,19 @@
         # observations to include more state.
         # See rllib/evaluation/observation_function.py for more info.
         "observation_fn": None,
+        # When replay_mode=lockstep, RLlib will replay all the agent
+        # transitions at a particular timestep together in a batch. This allows
+        # the policy to implement differentiable shared computations between
+        # agents it controls at that timestep. When replay_mode=independent,
+        # transitions are replayed independently per policy.
+        "replay_mode": "independent",
     },
 
+    # === Replay Settings ===
```

Comment:
This is very cool. We should be able to implement R2D2 then just setting this to >1.